io/net/ethtool/ethtool_tests.sh.data/README.txt

```diff
@@ -0,0 +1,14 @@
+Tests the network driver and interface with 'ethtool' command.
+12 different options of ethtool command are tested for each
```

Comment:
I don't see 12 tests in the test script. The number of tests depends on the multiplex file. Please clarify the multiplex usage instead.